contracts/scoring/dsf.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { IScoringFunction } from "../interface/ScoringFunction.sol";
+import { HashingHub } from "../HashingHub.sol";
+import { Hub } from "../Hub.sol";
+
+contract DSF is IScoringFunction {
```

Comment:
Curious, why DSF?

Reply:
> Curious, why DSF?

Default Scoring Function. Didn't come up with anything more original 😅

Reply:
Changed it to PLDSF = Polynomial Long Division Scoring Function